veniq/dataset_collection/dataflow/main.py

```diff
@@ -0,0 +1,64 @@
+import os
+from argparse import ArgumentParser
+
+import d6tcollect
+import d6tflow
+
+from veniq.dataset_collection.dataflow.collectEM import TaskFindEM
+from veniq.dataset_collection.dataflow.preprocess import TaskAggregatorJavaFiles
+
+d6tcollect.submit = False
+
+if __name__ == '__main__':
+    system_cores_qty = os.cpu_count() or 1
+    parser = ArgumentParser()
+    parser.add_argument(
+        "-d",
+        "--dir",
+        required=True,
+        help="File path to JAVA source code for methods augmentations"
+    )
+    parser.add_argument(
+        "-o", "--output",
+        help="Path for file with output results",
+        default='augmented_data'
+    )
+    parser.add_argument(
+        "--jobs",
+        "-j",
+        type=int,
+        default=system_cores_qty - 1,
+        help="Number of processes to spawn. "
+             "By default one less than number of cores. "
+             "Be careful to raise it above, machine may stop responding while creating dataset.",
+    )
+    parser.add_argument(
+        "-z", "--zip",
+        action='store_true',
+        help="To zip input and output files."
+    )
+    parser.add_argument(
+        "-s", "--small_dataset_size",
```

Comment:
what is this argument?

Reply:
this argument will be used in future to generate small dataset. It's its size